README.md

```diff
@@ -4,6 +4,36 @@
 
 This repository contains the source for the Google-maintained Tomcat [docker](https://docker.com) image.
 
+# Building the image
```

Comment:
Yes this part should belong to DEVELOPING.md, however, it might be interesting to keep it here before the first release, as it is a mandatory step.